pom.xml

```diff
@@ -545,6 +545,36 @@
           <artifactId>dependency-check-maven</artifactId>
           <version>4.0.2</version>
         </plugin>
+        <plugin>
+          <groupId>org.apache.maven.plugins</groupId>
+          <artifactId>maven-checkstyle-plugin</artifactId>
+          <version>3.0.0</version>
```

Comment:
Add the version to <pluginManagement> instead?

Reply:
@nkalmar it is already inside the "pluginManagement" block

Reply:
My bad.